vpr/src/device/rr_graph_view.h

```diff
@@ -0,0 +1,99 @@
+#ifndef _RR_GRAPH_OVERLAY_
```

Comment:
The class you define here is called `RRGraphView`. The header is calls `rr_graph_view.h`. so you want the include guard also be called something like `RR_GRAPH_VIEW_H_`

Also, avoid _leading_ underscores, as any identifiers and in particular macros with leading underscores are reserved for internal use in C/C++.

Reply:
Agree. Updated.